js/views/playingScreen.js

```diff
@@ -12,22 +12,23 @@ var React = require('react'),
     Watermark = require('../components/watermark'),
     ResizeMixin = require('../mixins/resizeMixin'),
     CONSTANTS = require('../constants/constants');
+    ViewControlsVr = require('../components/viewControlsVr'),
+    _ = require('underscore');
```

Comment:
underscore is not necessary in this file.

Reply:
There is _.isNumber() usage in this file..
Should we change this to use typeof X === "number", thenw e can remove _ ?

Reply:
fix